src/extension/wps-download/src/test/resources/org/geoserver/wps/gs/download/mapRemoteSimple11Opacity.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<wps:Execute version="1.0.0" service="WPS"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.opengis.net/wps/1.0.0"
+             xmlns:wfs="http://www.opengis.net/wfs" xmlns:wps="http://www.opengis.net/wps/1.0.0"
+             xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:gml="http://www.opengis.net/gml"
+             xmlns:ogc="http://www.opengis.net/ogc" xmlns:wcs="http://www.opengis.net/wcs/1.1.1"
+             xmlns:xlink="http://www.w3.org/1999/xlink"
+             xsi:schemaLocation="http://www.opengis.net/wps/1.0.0 http://schemas.opengis.net/wps/1.0.0/wpsAll.xsd">
+  <ows:Identifier>gs:DownloadMap</ows:Identifier>
+  <wps:DataInputs>
+    <wps:Input>
+      <ows:Identifier>bbox</ows:Identifier>
+      <wps:Data>
+        <wps:BoundingBoxData crs="EPSG:4326">
+          <ows:LowerCorner>-2.4 1.4</ows:LowerCorner>
+          <ows:UpperCorner>0.4 4.2</ows:UpperCorner>
+        </wps:BoundingBoxData>
+      </wps:Data>
+    </wps:Input>
+    <wps:Input>
+      <ows:Identifier>width</ows:Identifier>
+      <wps:Data>
+        <wps:LiteralData>256</wps:LiteralData>
+      </wps:Data>
+    </wps:Input>
+    <wps:Input>
+      <ows:Identifier>height</ows:Identifier>
+      <wps:Data>
+        <wps:LiteralData>256</wps:LiteralData>
+      </wps:Data>
+    </wps:Input>
+    <wps:Input>
+      <ows:Identifier>layer</ows:Identifier>
+      <wps:Data>
+        <wps:ComplexData>
+          <dwn:Layer xmlns:dwn="http://geoserver.org/wps/download">
+            <dwn:Capabilities>http://geoserver.org/geoserver/wms?request=GetCapabilities&amp;version=1.1.0</dwn:Capabilities>
+            <dwn:Name>cite:BasicPolygons</dwn:Name>
+            <dwn:Opacity>50</dwn:Opacity>
```

Comment:
I was expecting to see a partially transparent output, but it's not happening, both mapSimple.png and mapSimpleWithAlpha.png lack transparency, in that the alpha channel values are all 255.

In other words, picking a file as an example

```
aaime@colossus ~/devel/git-gs/src/extension/wps-download/src/test/resources/org/geoserver/wps/gs/download ((7e1cfd9515...)) $ gdalinfo -stats mapSimpleWithAlpha.png
Driver: PNG/Portable Network Graphics
Files: mapSimpleWithAlpha.png
Size is 256, 256
Image Structure Metadata:
  INTERLEAVE=PIXEL
Corner Coordinates:
Upper Left  (    0.0,    0.0)
Lower Left  (    0.0,  256.0)
Upper Right (  256.0,    0.0)
Lower Right (  256.0,  256.0)
Center      (  128.0,  128.0)
Band 1 Block=256x1 Type=Byte, ColorInterp=Red
  Minimum=0.000, Maximum=255.000, Mean=113.648, StdDev=126.471
  Mask Flags: PER_DATASET ALPHA 
  Metadata:
    STATISTICS_MAXIMUM=255
    STATISTICS_MEAN=113.64826965332
    STATISTICS_MINIMUM=0
    STATISTICS_STDDEV=126.47050543937
    STATISTICS_VALID_PERCENT=100
Band 2 Block=256x1 Type=Byte, ColorInterp=Green
  Minimum=0.000, Maximum=255.000, Mean=113.648, StdDev=126.471
  Mask Flags: PER_DATASET ALPHA 
  Metadata:
    STATISTICS_MAXIMUM=255
    STATISTICS_MEAN=113.64826965332
    STATISTICS_MINIMUM=0
    STATISTICS_STDDEV=126.47050543937
    STATISTICS_VALID_PERCENT=100
Band 3 Block=256x1 Type=Byte, ColorInterp=Blue
  Minimum=0.000, Maximum=255.000, Mean=216.206, StdDev=42.429
  Mask Flags: PER_DATASET ALPHA 
  Metadata:
    STATISTICS_MAXIMUM=255
    STATISTICS_MEAN=216.20637512207
    STATISTICS_MINIMUM=0
    STATISTICS_STDDEV=42.429166719178
    STATISTICS_VALID_PERCENT=100
Band 4 Block=256x1 Type=Byte, ColorInterp=Alpha
  Minimum=255.000, Maximum=255.000, Mean=255.000, StdDev=0.000
  Metadata:
    STATISTICS_MAXIMUM=255
    STATISTICS_MEAN=255
    STATISTICS_MINIMUM=255
    STATISTICS_STDDEV=0
    STATISTICS_VALID_PERCENT=100
```

See the stats for the alpha band (nr 4), min and max are both 255. With a transparency of 50%, I was expecting to see the alpha value at 122 instead (or 123...).

Reply:
Incorrect test file provided and the test was not properly detecting the mismatch (same file other than the opacity).  Correct file now provided and the test changed to test a pixel Alpha.